### PR TITLE
fix: typos

### PR DESCRIPTION
**Pull request by omahs**
fix: typos

### CODE REVIEW

**Review by ajsutton — APPROVED** [review]

**Review by tynes — APPROVED** [review]

**Comment by tynes** [comment]
@omahs Thank you for your contribution, it would be easier to get this merged if it was more PRs with less changes as the codeowners policy is difficult getting review from everybody

**Comment by refcell** [comment]
Closing due to no updates in a month (stale)

**Pull request closed without merging**
